runtime/core/patch/openfst/extensions/special/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+file(GLOB HEADER_FILES ../../include/fst/extensions/special/*.h)
```

Comment:
what is the function of this file?

Reply:
I just copy the file from this pr: https://github.com/kkm000/openfst/pull/36

Reply:
do we use it now?

Reply:
https://github.com/kkm000/openfst/pull/36/files#diff-d34b9d7d7cb7d326f283b20078ce57bcb89805249609fd257eccd3f069550b8bR38-R49

Please see the patch.